contracts/vortex/CarbonVortex.sol

```diff
@@ -625,11 +625,6 @@ contract CarbonVortex is ICarbonVortex, Upgradeable, ReentrancyGuardUpgradeable,
         // transfer the tokens to caller
         token.safeTransfer(msg.sender, targetAmount);
 
-        // if the target token is native, refund any excess native token to caller
```

Comment:
This obviously doesn't matter, due to the reetnrancy guard (but you already know that 😉)